### DISCUSSION TITLE
How to train your own model

### DISCUSSION THREAD

**Discussion by fontnet1** [idiap/coqui-ai-TTS#204] [Q&A] [answered]
I wanted to know if we can create our own model with your codes. I want to use the Persian dataset. Is it possible? Please provide a video that shows how to create a model with your codes, or if there is no video, introduce a site.

**Comment by eginhard** [comment, marked as answer]
See the documentation: https://coqui-tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html

**Comment by fontnet1** [comment]
Dear friend, I am new to this model and I don't know how to use it. Please introduce me to a video or a roadmap by reading this model and setting it up. How can I use this model and its codes?

**Comment by fontnet1** [comment]
Because I want to make a model for Persian.
